lcls_tools/common/measurements/screen_beam_profile_measurement.py

```diff
@@ -0,0 +1,99 @@
+from lcls_tools.common.devices.screen import Screen
+from lcls_tools.common.image.processing import ImageProcessor
+from lcls_tools.common.data.fit.projection import ProjectionFit
+from lcls_tools.common.measurements.measurement import Measurement
+import numpy as np
+
+
+class ScreenBeamProfileMeasurement(Measurement):
+    """
+    ScreenBeamProfileMeasurement class that allows for background subtraction and roi cropping
+    ------------------------
+    Arguments:
+    name: str (name of measurement default is beam_profile),
+    device: Screen (device that will be performing the measurement),
+    image_processor: ImageProcessor ()
+    fitting_tool: ProjectionFit ()
+    fit_profile: bool = True ()
+    ------------------------
+    Methods:
+    single_measure: measures device and returns raw and processed image
+    measure: does multiple measurements and has an option to fit the image profiles
+    """
+
+    name: str = "beam_profile"
+    device: Screen
+    image_processor: ImageProcessor
+    fitting_tool: ProjectionFit
+    fit_profile: bool = True
+    # charge_window: Optional[ChargeWindow] = None
+
+    def measure(self, n_shots: int = 1) -> dict:
+        """
+        Measurement function that takes in n_shots as argumentment, where n_shots is the number of
+        image profiles (is this what I want to call it?)
```

Comment:
argumentment -> argument. I think `images', is the right term here vs. profiles. We're taking n pictures of the beam.